src/xtgeo/surface/_regsurf_oper.py

```diff
@@ -155,11 +184,13 @@ def distance_from_point(self, point=(0, 0), azimuth=0.0):
     self.set_values1d(svalues)
 
 
-def get_value_from_xy(self, point=(0.0, 0.0)):
+def get_value_from_xy(self, point=(0.0, 0.0), sampling="bilinear"):
     """Find surface value for point X Y."""
 
     xcoord, ycoord = point
 
+    option = 0 if sampling == "bilinear" else 2
```

Comment:
Pick integer values  for `option` from a dictionary instead? Will make it easier to extend to more interpolations later and make the code more readable.

Reply:
Will consider